README.md

```diff
@@ -18,7 +18,7 @@ orbs:
 jobs:
   build:
     docker:
-      - image: circleci/node:7.10
+      - image: cimg/node:16.13.2
```

Comment:
Suggested version of node being 16.13.2 which is latest LTS version at time of writing